Test/git-issues/git-issue-590.dfy

```diff
@@ -1,3 +1,3 @@
-// RUN: %dafny /compile:3 x.cs > "%t"
+// dafny should emit exit value 1
+// RUN: %dafny /compile:3 x.cs > "%t" || echo ERROR EXIT >> "%t"
```

Comment:
Does `||` work on Windows?

Reply:
Aargh no it won't when we combine with the expanded CI tests. I'll change countVerificationErrors to suppress command-line errors as well -- and hope Robin can finish the test infrastructure soon